### ISSUE TITLE
[🚀 Feature]: The overview page of Selenium should be written better and without assuming it will be used for test

### ISSUE THREAD

**Issue by rpallavisharma** [SeleniumHQ/seleniumhq.github.io#1397]
### Feature and motivation

This page- https://www.selenium.dev/documentation/overview/

**Comment by asolntsev** [comment]
@pallavigitwork Isn't it duplicate of https://github.com/SeleniumHQ/seleniumhq.github.io/issues/1369?

**Comment by titusfortner** [comment]
So, we do have an example of a Selenium script without it being a test here: https://www.selenium.dev/documentation/

It's in the "Hello Selenium" files: (e.g., https://github.com/SeleniumHQ/seleniumhq.github.io/blob/trunk//examples/java/src/test/java/dev/selenium/hello/HelloSelenium.java)

Maybe we can just link to that at the top of the "first script" page?

I guess I'm more interested in helping people build out something more extensive rather than just create a standalone file with a script. If you want a standalone file it's easy enough to copy/paste everything there. I'd rather put it in a more usable context for the majority of users.

**Comment by rpallavisharma** [comment]
> @pallavigitwork Isn't it duplicate of #1369?

no. replied on the 1367 thread. one was for the page content, other for code.

**Comment by titusfortner** [comment]
@pallavigitwork what do you think of this: #1479 

It changes First Script examples into standalone files and removes the assertions
Then it adds a new page on Running Tests that tries to put the code into context of how to use it with a larger project.

**Comment by rpallavisharma** [comment]
i believe this PR handled it - https://github.com/SeleniumHQ/seleniumhq.github.io/pull/1479
